src/main/java/gov/nasa/pds/tools/validate/rule/pds4/FileReferenceValidationRule.java

```diff
@@ -430,9 +430,11 @@ private boolean validateFileAreaDefinitionAndContent(String fileName, TinyNodeIm
             // Is is possible that there's no corresponding problemType. Must check for
             // null-ness before calling checkGenericDocument() function.
             if (problemType == null) {
-              LOG.error(
-                  "FileReferenceValidationRule:Cannot retrieve ProblemType from provided doctype {}",
-                  doctype);
+            	if (documentStandardId != null || !"".equalsIgnoreCase(doctype)) {
+            		LOG.error(
+            				"FileReferenceValidationRule:Cannot retrieve ProblemType from provided doctype {}",
+            				doctype);
+            	}
```

Comment:
This change may be not be needed/desired but it seems correct from deeper code that if a problemType is defined when doctype is "" then it always returns true (hard coded) if documentStandardId is null. Simpler doing it here than dealing with the fallout of creating a problemType.